src/main/kotlin/org/veupathdb/service/eda/compute/EDA.kt

```diff
@@ -177,7 +177,7 @@ object EDA {
       }
     }
 
-    // Return the job ID with no-such-job status
+    // Make a new job response and set values from the job just submitted.
```

Comment:
why this comment change?

Reply:
Oops.  Changed the wrong comment.  Meant to change on 173.  Committing a fix.